lib/index_group.rb

```diff
@@ -152,7 +166,48 @@ def name_pattern
 
     def delete(index_name)
       logger.info "Deleting index #{index_name}"
-      @client.indices.delete(index: index_name)
+      @long_timeout_client.indices.delete(index: index_name)
+    end
+
+    # Sort the index map by date, remove the most recent two (active and latest inactive) and return hash
+    def cleaned_aliases
```

Comment:
I'd love some tests for this in general, but especially around ensuring that we discard the correct indices from the list of those that we want to delete 😱

Reply:
Honestly I wasn't too happy with this logic myself the more I thought about it, so was holding back on tests until I had some comment one way or the other as it could change how the test logic is written.

I'm going to put some guards in here. It'll make everything so much safer!